tools/ci/inria/other-configs/script

```diff
@@ -39,6 +39,6 @@ ${main} -conf --disable-native-compiler \
         -no-native
 ${main} -conf --disable-flat-float-array
 ${main} -conf --enable-flambda
-${main} -conf --enable-frame-pointers -conf --enable-reserved-header-bits=27
+${main} -conf --enable-frame-pointers -conf --reserved-header-bits=26
```

Comment:
out of curiosity, why move from 27 to 26 here?

Reply:
This is a typo ("26" is the canonical value in various resources I found, including old mlvalues.h comments). Well caught!

Reply:
(also the change in the switch name is incorrect. My understanding of autoconf developed rapidly today, and is still quite rudimentary).